site/content/en/docs/handbook/addons.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Minikube Addons"
+weight: 7
+description: >
+  List of addons for minikube
+---
+
+List of addons can be found here - [list of addons](https://github.com/kubernetes/minikube/tree/master/deploy/addons)
```

Comment:
use document auto generate here (to automaticly list addons list from minikube)

Reply:
yes, I plan to tackle that next. Right now, I am having an issue where the addons page does not show up on the site. The link to the addons page yields a 404. All I did was I added a addons.md page in `docs/handbook`. Do I need to do anything else? Like `hugo new addons.md` or something?

Reply:
actually no need to use auto -generate here. never mind

Reply:
maybe that page was missing I see one here https://minikube.sigs.k8s.io/docs/handbook/deploying/